examples/record/Record.hs

```diff
@@ -0,0 +1,17 @@
+import Extensible
+import qualified RecordBase as R
+import qualified RecordBase
+
+
+data WithString
+
+R.extendRec "Rec" [] [t|WithString|] $
+  R.defaultExtRec {
+    R.typeR1 = Ann ("label1", [t|String|]),
+    R.typeR2 = Ann ("label2", [t|String|]),
+    R.typeRecX = [("R3", "contents", [t|Int|])]
```

Comment:
Can the contents also have record fields, or not?

Reply:
I'm not sure what you mean, sorry

In this case the extension constructor comes out to `R3 {contents :: Int}`

Reply:
Ah, I see - can there be multiple fields (by using a 5-tuple, I suppose, for two fields)?